helm/values.yaml

```diff
@@ -155,3 +155,20 @@ servicemonitor:
   interval: 15s
   secure: false
   tlsConfig: {}
+
+compatibilityJob:
+  enabled: false
+  name: schema-registry-compatibility-job
+  hook: post-install, pre-upgrade
```

Comment:
```suggestion
  hook: post-install, post-upgrade
```

Reply:
Could you explain how using pre vs post would differ for upgrade?